mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -0,0 +1,66 @@
+import { FormLabel } from '@mui/material'
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { FormQuestionDiv, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import ButtonSubmit from '../ButtonSubmit'
+import CheckboxGroupMangroveWithLabel from '../CheckboxGroupMangroveWithLabel'
+import restorationAimsQuestionsAndAnswers from './restorationAimsQuestionsAndAnswers'
+
+const RestorationAimsForm = () => {
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const reactHookFormInstance = useForm()
+
+  const { handleSubmit: validateInputs } = reactHookFormInstance
+
+  const handleSubmit = () => {
+    setIsSubmitting(true)
+    // hit api here
+    setIsSubmitting(false)
+  }
+  const generateOptionsObjectFromAnswers = (answers) =>
+    answers.map((answer) => ({
+      label: answer,
+      value: answer
+    }))
+  const ecologicalAimsOptions = generateOptionsObjectFromAnswers(
+    restorationAimsQuestionsAndAnswers[1].answers
+  )
+  const socioEconomicAimsOptions = generateOptionsObjectFromAnswers(
+    restorationAimsQuestionsAndAnswers[2].answers
+  )
+  const otherAimsOptions = generateOptionsObjectFromAnswers(
```

Comment:
interesting! have a totally different approach in section 2, but maybe we can discuss each one. I pushed my branch up, will create a draft PR so anyone can view :)

Reply:
Cool, Ill take a look.